src/index.js

```diff
@@ -0,0 +1,18 @@
+import "./styles.css";
```

Comment:
I had planned to move this to the `js` file as part of the clean up stage, since that seems to be the file your using as the main source file.

Reply:
Okay Part 2 is up, but it's a PR on my fork because I'm aiming at my branch which obviously isn't in your repo.
https://github.com/Irev-Dev/CascadeStudio/pull/7